caustic/lenses/epl.py

```diff
@@ -6,7 +6,7 @@
 from ..cosmology import Cosmology
 from ..utils import derotate, translate_rotate
 from .base import ThinLens
-from ..parametrized import unpack
+from ..sims import unpack
```

Comment:
I'm open to returning this to how it was, but I did it because in my head the parametrized system is built specifically to make simulators work so it made sense to me to package all the simulator framework stuff together. But I see how the core parametrize framework is conceptually different from our pre-built user friendly simulators. So another option would be to put them in their own folder like `params` and keep it separate.